sources/packages/backend/apps/queues/test/app.e2e-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Test, TestingModule } from "@nestjs/testing";
+import { INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { QueuesModule } from "./../src/queues.module";
+
+describe("QueuesController (e2e)", () => {
+  let app: INestApplication;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [QueuesModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
+  it("/ (GET)", () => {
+    return request(app.getHttpServer())
```

Comment:
This does not seems to be a test that would pass. I would recommend to remove it.